provision/cluster/metrics.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 tsuru authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cluster
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/tsuru/tsuru/log"
+	"github.com/tsuru/tsuru/servicemanager"
+)
+
+var (
+	desc        = prometheus.NewDesc("tsuru_cluster_info", "Basic information about existing clusters", []string{"provisioner", "name"}, nil)
+	failureDesc = prometheus.NewDesc("tsuru_cluster_fetch_fail", "indicates whether failed to get clusters", []string{}, nil)
```

Comment:
Is it a best practice count the fails to obtain other metric? I really don't know (I've no metrics design experience with Prom :/).